pkg/securitycontext/provider.go

```diff
@@ -95,3 +96,53 @@ func makeCapabilites(capAdd []api.Capability, capDrop []api.Capability) ([]strin
 	}
 	return addCaps, dropCaps
 }
+
+func determineEffectiveSecurityContext(pod *api.Pod, container *api.Container) *api.SecurityContext {
```

Comment:
Hold up.  @smarterclayton I thought we (you) made a string case that every field in container.SecurityContext had "do the right thing if not provided" semantics.  I took a look at it, and (while the comments are obsolete and terrible) that seems to be the case.

In this case DTRT means defer to PSC, doesn't it?  Why do I find myself convincing you when you convinced me?